### DISCUSSION TITLE
「塗り」のグラデーション対応

### DISCUSSION THREAD

**Discussion by toruki** [kyubuns/AkyuiUnity#29] [Q&A] [answered]
「塗り」のグラデーションに対応することは難しいでしょうか？

**Comment by kyubuns** [comment]
グラデーション対応は常々入れたいなと思ってるので、そろそろやります！

**Reply by toruki** [reply]
ありがとうございます！
期待して待ってます。

**Comment by kyubuns** [comment, marked as answer]
v1.1.16にてLinearGradient, RadialGradientに対応しました。
AngularGradientはAdobeXDのバグを踏んだので治り次第対応します。

**Reply by toruki** [reply]
期待どおりに動作することを確認しました。
迅速な対応ありがとうございます。
